setup.sh

```diff
@@ -15,11 +15,10 @@
 # limitations under the License.
 
 # Description:
-# bash setup.sh MODE={stable,nightly}
+# bash setup.sh MODE={stable,nightly,pinned} DEVICE={tpu,gpu}
```

Comment:
@sizhit2 Can we add a comment stating that not all combinations are supported, e.g., TPU and pinned are not supported?

Reply:
Sorry for the confusion. We don't support GPU pinned mode at this moment. Let me delete the pinned mode here.